xchange-core/src/main/java/org/knowm/xchange/service/polling/trade/PollingTradeService.java

```diff
@@ -113,14 +129,21 @@ public boolean cancelOrder(String orderId)
    * @see #createTradeHistoryParams()
    * @see org.knowm.xchange.service.polling.trade.params.TradeHistoryParamsAll
    */
-  public UserTrades getTradeHistory(TradeHistoryParams params) throws IOException;
+  UserTrades getTradeHistory(TradeHistoryParams params) throws IOException;
 
   /**
    * Create {@link TradeHistoryParams} object specific to this exchange. Object created by this method may be used to discover supported and required
    * {@link #getTradeHistory(TradeHistoryParams)} parameters and should be passed only to the method in the same class as the createTradeHistoryParams
    * that created the object.
    */
-  public TradeHistoryParams createTradeHistoryParams();
+  TradeHistoryParams createTradeHistoryParams();
+
+  /**
+   * Create {@link OpenOrdersParams} object specific to this exchange. Object created by this method may be used to discover supported and required
+   * {@link #getOpenOrders(OpenOrdersParams)} parameters and should be passed only to the method in the same class as the createOpenOrdersParams
+   * that created the object.
+   */
+  OpenOrdersParams createOpenOrdersParams();
```

Comment:
I think it might actually be a good thing. It forces the implementor, not the user of the exchange to think things out. in the end it's easier for the end user. What do you think?